IPlug/CLAP/IPlugCLAP.cpp

```diff
@@ -441,18 +474,23 @@ void IPlugCLAP::ProcessInputEvents(const clap_input_events *in_events) noexcept
           ISysEx sysEx(event->time, midiSysex->buffer, midiSysex->size);
           ProcessSysEx(sysEx);
           //mSysExDataFromProcessor.Push(sysEx);
+          break;
         }
           
         case CLAP_EVENT_PARAM_VALUE:
         {
           auto paramValue = ClapEventCast<clap_event_param_value>(event);
           
           int paramIdx = paramValue->param_id;
-          double value = paramValue->value;
-          
-          GetParam(paramIdx)->Set(value);
-          SendParameterValueFromAPI(paramIdx, value, false);
-          OnParamChange(paramIdx, EParamSource::kHost, event->time);
+          if (paramIdx >= 0 && paramIdx < this->NParams())
```

Comment:
Why is this check needed?

Reply:
I got some crazy (obviously uninitialized) values from Bitwig 4.3 so I thought it might be a good idea... ;-)

Reply:
I'm unclear on why the paramIdx should be out of range - it's also possible that we should be reporting this as host misbehaviour if it does happen, but I've not seen it to date - I would expect it to crash the plugin if it did happen.

Reply:
From what I recall the params are reported and the ids are arbitrary, so the host should just pass back values that have been passed to it from the plugin.